modules/sdk/ant-bnd/src/main/java/com/liferay/ant/bnd/resource/bundle/AggregateResourceBundleLoaderAnalyzerPlugin.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ant.bnd.resource.bundle;
+
+import aQute.bnd.header.Attrs;
+import aQute.bnd.header.Parameters;
+import aQute.bnd.osgi.Analyzer;
+import aQute.bnd.osgi.Constants;
+import aQute.bnd.service.AnalyzerPlugin;
+
+import aQute.libg.filters.AndFilter;
+import aQute.libg.filters.LiteralFilter;
+import aQute.libg.filters.NotFilter;
+import aQute.libg.filters.SimpleFilter;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Gregory Amerson
+ */
+public class AggregateResourceBundleLoaderAnalyzerPlugin
+	implements AnalyzerPlugin {
+
+	@Override
+	public boolean analyzeJar(Analyzer analyzer) throws Exception {
+		Parameters aggregateResourceBundlesParameters = new SortedParameters(
+			analyzer.getProperty("-aggregateresourcebundles"));
+
+		if (aggregateResourceBundlesParameters.size() == 0) {
+			return false;
+		}
+
+		Parameters provideCapabilityHeaders = new SortedParameters(
+			analyzer.getProperty(Constants.PROVIDE_CAPABILITY));
+
+		Parameters requireCapabilityHeaders = new SortedParameters(
+			analyzer.getProperty(Constants.REQUIRE_CAPABILITY));
+
+		List<String> aggregateKeys = new ArrayList<>(
+			aggregateResourceBundlesParameters.keySet());
+
+		Collections.sort(aggregateKeys);
```

Comment:
Yes i believe we can remove this line.  That was in there from before I created the SortedParameters class.  This should be good to remove.